apps/wpcom-block-editor/src/common/override-close-btn.js

```diff
@@ -0,0 +1,27 @@
+/* global calypsoifyGutenberg */
+
+/**
+ * External dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+import { get } from 'lodash';
+
+domReady( () => {
+	const closeButtonInception = setInterval( function() {
```

Comment:
Would it be possible to follow up in another PR avoiding the setInterval here? Feels like we should be able to listen to some Gutenberg redux state instead.